src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,62 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
 public class SalaryInfo {
+    public static final String DATE_FORMAT = "d.MM.yyyy";
+    public static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern(DATE_FORMAT);
+    public static final String DATA_SEPARATOR = " - ";
+    public static final String COLUMN_SEPARATOR = " ";
+    public static final String REPORT_HEADER = "Report for period ";
+    public static final int DATE_COLUMN_INDEX = 0;
+    public static final int NAME_COLUMN_INDEX = 1;
+    public static final int QOTY_COLUMN_INDEX = 2;
+    public static final int SUM_COLUMN_INDEX = 3;
+
+    private int getTotalSalaryByName(String[] data, String name,
```

Comment:
Usually, the private method is placed after the public ([Ordering of class contents](https://mate-academy.github.io/style-guides/java/java.html#s3.4.2-ordering-class-contents))